src/application/services/useNoteSettings.ts

```diff
@@ -0,0 +1,47 @@
+import { ref, type Ref } from 'vue';
+import type NotesSettings from '@/domain/entities/NotesSettings';
+import { noteSettingsService } from '@/domain';
+
+/**
+ * Note settings hook state
+ */
+interface UseNoteSettingsComposableState {
+  /**
+   * NoteSettings ref
+   */
+  noteSettings: Ref<NotesSettings | null>;
+
+  /**
+   * Load note settings
+   *
+   * @param publicId - note publicId
+   */
+  loadSettings: (publicId: string) => Promise<void>;
+}
+
+
+/**
+ * Application service for working with the Note settings
+ */
+export default function (): UseNoteSettingsComposableState {
+  /**
+   * NoteSettings ref
+   */
+  const noteSettings = ref<NotesSettings | null>(null);
+
+  /**
+   * Get note settings
+   *
+   * @param publicId - Note publicId
+   */
+  const loadSettings = async (publicId: string): Promise<void> => {
```

Comment:
it's better not to use the "public" prefix, because from the client-perspective there is only "id" property. It doesn't know about internal id which is a backend feature.
```suggestion
  const loadSettings = async (id: string): Promise<void> => {
```